diff --git a/lib/server/utils/network.js b/lib/server/utils/network.js
--- a/lib/server/utils/network.js
+++ b/lib/server/utils/network.js
@@ -69,7 +69,6 @@ function httpRequest(params, onResponse, onError) {
     } else {
       parsedProxyURL.secureProxy = parsedProxyURL.protocol === "https:";
       options.agent = new import_utilsBundle.HttpsProxyAgent(parsedProxyURL);
-      options.rejectUnauthorized = false;
     }
   }
   let cancelRequest;
